package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
+    "sass": "^1.55.0",
```

Comment:
not being utilized from the looks of it, might be worth doing uninstalling to clean up files.

Reply:
Deleted from app. Thanks!